.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
This seems logical, since `go fmt` produces only LF endings and if the `.go` files get checked out on windows with CRLF, there are "every file changed" when running `go fmt`.